plugins/reach/lib/api/data/entryVendorTaks/KalturaQuizVendorTaskData.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @package plugins.reach
+ * @subpackage api.objects
+ * @relatedService EntryVendorTaskService
+ */
+class KalturaQuizVendorTaskData extends KalturaVendorTaskData
+{
+	/**
+	 * Number Of Questions.
+	 *
+	 * @var int
+	 */
+	public $numberOfQuestion;
```

Comment:
i would change it is Questions and not Question, just confusing with specific question

Reply:
done